src/notyf.view.ts

```diff
@@ -211,6 +221,16 @@ export class NotyfView {
     return elem;
   }
 
+  private _isHTMLElement (element: any): element is HTMLElement {
```

Comment:
I appreciate the intent here, but I think this is expensive, even more coming from a "lightweight" library like Notyf. Also, it has side effects, because if the element happens to be in the DOM, it gets removed. Can you find a cheaper solution for this check?